### PR TITLE
feat(Tebex): add presence

### PR DESCRIPTION
**Pull request by DarkVillager**
## Description 

Added tebex & store's with Buycraft/Tebex in it's domain
## Acknowledgements
- [X] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [X] I linted the code by running `yarn format`
- [X] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

File: websites/T/Tebex/metadata.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "Dark_Ville",
+		"id": "638080361179512853"
+	},
+	"service": "Tebex",
+	"description": {
+		"en": "Fund your game server costs with Tebex. The leading payment plugin for Minecraft, Rust, ARK, GTA V and more.",
+		"nl": "Financier jouw game server kosten met Tebex. De vooraanstaande betalingsplugin voor Minecraft, Rust, ARK, GTA V en meer."
```
Comment (EncryptedDev) [inline websites/T/Tebex/metadata.json:10]: @Bas950

File: websites/T/Tebex/presence.ts [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+							url: href,
+						},
+					];
+					break;
+				}
+				case "games": {
+					presenceData.details = "Viewing all games";
+					presenceData.buttons = [
+						{
+							label: "View All Games",
```
Comment (EncryptedDev) [inline websites/T/Tebex/presence.ts:43]: Sounds really repetitive with `details` possibly `View Page` instead

File: websites/T/Tebex/presence.ts [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+							url: href,
+						},
+					];
+					break;
+				}
+				case "partners": {
+					presenceData.details = "Viewing partners";
+					presenceData.buttons = [
+						{
+							label: "View Partners",
```
Comment (EncryptedDev) [inline websites/T/Tebex/presence.ts:63]: Ditto

File: websites/T/Tebex/presence.ts [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+							url: href,
+						},
+					];
+					break;
+				}
+				case "contact": {
+					presenceData.details = "Viewing contact info";
+					presenceData.buttons = [
+						{
+							label: "View The Contact Page",
```
Comment (EncryptedDev) [inline websites/T/Tebex/presence.ts:73]: Not really needed

File: websites/T/Tebex/presence.ts [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+				presenceData.state = search?.value;
+				presenceData.smallImageKey = Assets.SearchImage;
+			} else {
+				presenceData.details = `Reading the ${
+					document.querySelector('[data-testid="page.title"]')?.textContent
+				} page`;
+				presenceData.smallImageKey = Assets.ReadingImage;
+				presenceData.buttons = [
+					{
+						label: "Read The Docs",
```
Comment (EncryptedDev) [inline websites/T/Tebex/presence.ts:108]:
```suggestion
						label: "Read Docs",
```

File: websites/T/Tebex/presence.ts [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+			}'s`;
+			switch (pathnameSplit[1]) {
+				case "": {
+					presenceData.details = `${viewing} homepage`;
+					break;
+				}
+				case "category": {
+					presenceData.buttons = [
+						{
+							label: "View Category",
```
Comment (EncryptedDev) [inline websites/T/Tebex/presence.ts:128]: Not really needed

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/T/Tebex/metadata.json (hunk 1 above)
Comment (DarkVillager) [reply websites/T/Tebex/metadata.json:10]: @QkeleQ10

**Review by EncryptedDev — APPROVED** [review]
lgtm once bas approves translation

**Review by Bas950 — APPROVED** [review]

**Pull request merged**